### PR TITLE
Add ClearSecuredValuesSerializer annotation to all sensitive bean fields

### PR DESCRIPTION
**Pull request by LEQADA**
## Description
Customer related data shouldn't be logged. Therefore we need to provide a switch in the configuration which allows to anonymise/de-anonymise those data.
This switch implemented as an environment variable `HIDE_CUSTOMER_PERSONAL_DATA` which can be in 2 states: `"true"` OR `"false"`.
Default state is `"true"`. So if the environment variable is not provided then customer related data will be replaced by `<HIDDEN>` placeholder.

### CODE REVIEW

**Comment by LEQADA** [comment]
@butenkor you mean this one?
https://github.com/commercetools/commercetools-payone-integration/pull/224/files#diff-890fee622c7936209141adf0aa0b2988

File: service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+    @Test
+    public void createsMapWithHiddenSecrets() {
+        assertThat(request.toStringMap(true)).containsOnly(
+                entry("narrative_text", ClearSecuredValuesSerializer.PLACEHOLDER),
+                entry("param", ClearSecuredValuesSerializer.PLACEHOLDER),
+                entry("key", ClearSecuredValuesSerializer.PLACEHOLDER),
+                entry("clearingtype", clearingType),
+                entry("errorurl", errorUrl),
+                entry("vatid", vatId),
```
Comment (lojzatran) [inline service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequestTest.java]: What about this VAT ID? Isn't it also personal info?

Comment (LEQADA) [reply service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequestTest.java]: @floriansattler ?

**Comment by LEQADA** [comment]
@lojzatran there is https://github.com/commercetools/commercetools-payone-integration/blob/0997157406be6bf16bc77eeac79256c26fa17367/service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/banktransfer/BankTransferRequestTest.java

File: service/src/main/java/com/commercetools/pspadapter/payone/util/ClearSecuredValuesSerializer.java [hunk 2]
```diff
@@ -31,7 +40,11 @@ private ClearSecuredValuesSerializer(final BeanProperty property) {
 
     @Override
     public void serialize(final String value, final JsonGenerator gen, final SerializerProvider serializers) throws IOException, JsonProcessingException {
```
[the remark this answers is not in the document]
Comment (LEQADA) [reply service/src/main/java/com/commercetools/pspadapter/payone/util/ClearSecuredValuesSerializer.java]: https://github.com/commercetools/commercetools-payone-integration/pull/224/commits/0997157406be6bf16bc77eeac79256c26fa17367

File: service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequestTest.java (hunk 1 above)
Comment (floriansattler) [reply service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequestTest.java]: It`s a public info, as it is from companies.

**Pull request merged**